packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -2408,6 +2408,24 @@ void main() {
     androidIdentifier: 'com.example.flutter_project');
   });
 
+  testUsingContext('plugin includes native unit tests', () async {
+    Cache.flutterRoot = '../..';
+
+    final CreateCommand command = CreateCommand();
+    final CommandRunner<void> runner = createTestCommandRunner(command);
+
+    await runner.run(<String>['create', '--no-pub', '--template=plugin', '--platform=ios', projectDir.path]);
+
+    expect(projectDir
+        .childDirectory('example')
+        .childDirectory('ios')
+        .childDirectory('RunnerTests')
+        .childFile('FlutterProjectPluginTest.swift'), exists);
```

Comment:
```
03:49 +167 ~1 -1: test/commands.shard/permeable/create_test.dart: plugin includes native unit tests [E]                                                                                                
  Expected: a file system entity that exists
    Actual: LocalFile:<LocalFile: '/b/s/w/ir/x/t/flutter_tools.CSBNSE/flutter_tools_create_test.OVGLNS/flutter_project/example/ios/RunnerTests/FlutterProjectPluginTest.swift'>
     Which: does not exist
```

Reply:
Sorry, I had roughed out the test but not actually run it yet, then forgot it was in there when I posted the draft for feedback.